console/src/components/UserSelectionPopup/UserSelectionPopup.js

```diff
@@ -42,7 +42,14 @@ const UserSelectionPopupBase = kind({
 		}
 	},
 
-	render: ({userId, usersList, updateUser, resetUserSettings, resetAll, ...rest}) => (
+	handlers: {
+		onResetAll: (ev, {onReset, resetAll}) => {
```

Comment:
Let's use `onResetAll` as the prop instead of `onReset` to help disambiguate from the single user reset features.